lib/modules/manager/gradle/extract.spec.ts

```diff
@@ -6,7 +7,8 @@ import { extractAllPackageFiles } from '.';
 
 jest.mock('../../../util/fs');
 
-function mockFs(files: Record<string, string>): void {
+// eslint-disable-next-line jest/no-export
```

Comment:
I didn't want to duplicate this snippet, so I exported it against the advice from my IDE..

Reply:
Please don't do this. The mocked implementation for `fs.getFileContentMap` is only needed for `extractAllPackageFiles()` in `extract.ts`, not for the Gcv plugin. The mock for `fs.getSiblingFileName` is there since tests for `apply from` refer to different sibling files.

It should be enough to simply extend Gcv tests with statements analogous to this: `fs.getSiblingFileName.mockReturnValueOnce('versions.lock');`

Reply:
Truns out the use of mockFs in gcv test file was dead code, we don't need it along with the export.